repo/blob/storage.go

```diff
@@ -17,6 +17,14 @@ var ErrSetTimeUnsupported = errors.Errorf("SetTime is not supported")
 // ErrInvalidRange is returned when the requested blob offset or length is invalid.
 var ErrInvalidRange = errors.Errorf("invalid blob offset or length")
 
+// TokenExpiredErrStr is the error string returned by the provider
+// when a token has expired.
+const TokenExpiredErrStr = "The provided token has expired"
```

Comment:
nit: does this need to be public?

Reply:
Yes. The const is defined in the blob package . It is used in two places: 
-  In the same blob package with the const ErrInvalidCredentials 
- In the s3 package, in the function added in this PR isInvalidCredentials()